wpeverest/everest-forms
# Fix -  Global Setting Premium Sidebar Design Issue.

### All Submissions:






### Changes proposed in this Pull Request:


This PR fixes:-
Remove the sliding effect in settings sidebar. 
Design issue in small screen.

### How to test the changes in this Pull Request:

1. Go to global setting
2. check the design issue and flickering issue is fixed or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix -  Global Setting Premium Sidebar Design Issue.
